refactors.txt

```diff
@@ -0,0 +1,14 @@
+Possible Refactors:
+  1.  make method names clearer/more succinct
+      - currently has long method names for BookingSystem class
+      - unsure if it makes it unclear as to the single responsibility of the method
+  2.  use keyword arguments for constructors, methods that require lengthy arguments
+      - currently, all classes take in either positional arguments only or a mix of positional and keyword (but keyword is only for default values or to load previously existng data)
+      - in future changes, make it explicit what you are asking for
+  3.  consolidate redundant code into separate method or class
+      - error validation seems to be everywhere
```

Comment:
I think this is a super interesting observation -- I don't think we have the tools yet to think about what design patterns to apply to deal with this yet, but definitely a good observation to make.